apps/asap-server/src/entities/research-output.ts

```diff
@@ -133,6 +136,7 @@ const isSharingStatus = (
   (sharingStatuses as ReadonlyArray<string>).includes(status);
 
 const isResearchOutputType = (type: string): type is ResearchOutputType =>
+  type !== 'Proposal' &&
```

Comment:
This seems strange. Why are proposals not a research output type? If proposals are remapped and never exposed to the frontend can we remove this type from the modal?